qa/selenium-ide/web-server/testCases/Vehicles

```diff
@@ -84,3 +84,4 @@
 </tbody></table>
 </body>
 </html>
+<!--covers test TST-18-->
```

Comment:
Per slack, let's extend the name="" attribute to include the test ID. That will allow us to support multiple tickets in the same file.